src/components/message/threadAttachment/gitHubAttachment.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import React, { useState, useEffect } from 'react';
+import path from 'path';
+// $FlowFixMe
+import parseGithubUrl from 'parse-github-url';
+import { Loading } from 'src/components/loading';
+import { Container, Column, GitHubBadge } from './style';
+import { ThreadTitle } from 'src/components/inboxThread/style';
+import { InnerMessageContainer } from '../style';
+
+const GitHubAttachment = (props: { url: string }) => {
+  const { url } = props;
+  const [loading, setLoading] = useState(true);
+  const [apiData, setApiData] = useState(null);
+  const [urlData] = useState(parseGithubUrl(url));
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const apiUrl = path.join('https://api.github.com/repos/', urlData.path);
+      const result = await fetch(apiUrl);
+      const json = await result.json();
+      setApiData(json);
+      setLoading(false);
+    };
+
+    fetchData();
+  }, []);
+
+  let attachment;
+
+  if (loading || !apiData) {
+    attachment = (
+      <Container style={{ padding: '16px 12px' }}>
+        <Loading />
+      </Container>
```

Comment:
We will definitely need a fallback here in case the API call fails instead of showing the loading indicator forever.

Reply:
I've added a fallback that just spits out the original url if there is a timeout or any bad request